metadata_finds/metadata_finds.sh

```diff
@@ -0,0 +1,209 @@
+                while IFS= read -r line;
+                do
+                        echo "Do you  want to  add $line database to the investigation? [y/n] (lower case please)"
+                        read sim </dev/tty
+                        if [[ $sim == "y" ]];
+                                then
+                                        echo "Adding..."
+                                        db_list[idx]=$line;
+                                        idx=$((idx+1)) 
+                        fi
+                done < "$file"
+
+                #unset all default databases from array
+		i_schm="information_schema"
+                p_schm="performance_schema"
+		msql="mysql"
+		daBase="Database"
+                sys="sys"
+                new_schema="new_schema"
+
+		index=0
+		for i in ${db_list[@]}; do
+                        if [ "$i" = "$i_schm" ] || [ "$i" = "$p_schm" ] || [ "$i" = "$msql" ] || [ "$i" = "$daBase" ] || [ "$i" = "$sys" ] || [ "$i" = "$new_schema" ]; then
+                                unset db_list[$index]            
+                        fi 
+                        let index++
+                done
+
+elif [ $all_db_anserwer == "n" ];
+	then
+    		#Loading  the databases names
+   		echo  "How mamy data bases you want to  identify the metadata?"
+    		read db_num
+
+    		index=1
+    
+    		while [ $index -le $db_num ]
+    		do
+   			echo "Type the database name$index: "
+    			read database
+    			db_list[index]=$database;
+    			index=$((index+1)) 
+    		done
+
+else
+    	echo "Please type y or n"
+	exit 1
+fi    
+echo "Here is the list I will be working on ${db_list[@]}"
+echo ""
+echo "Do you which to continue... [y/n] (lower case please)"
+read continuar
+
+if [ $continuar == "n" ]
+        then
+                exit 1
+fi
+declare -A queries
+
+####  the bellow  queries  will used to run in mentioned partners databases  ####
+#--Get List of all programs
+queries[0,0]="SELECT program_id,name,retired program_retired,uuid FROM program ORDER BY program_id"
+queries[0,1]="all-programs"
+
+#--Get all the workflow of the programs
+queries[1,0]="SELECT p.program_workflow_id,p.program_id,p.retired program_workflow_retired, p.concept_id,c.name,p.uuid program_workflow_uuid,c.uuid concept_uuid FROM program_workflow p INNER JOIN concept_name c ON p.concept_id=c.concept_id ORDER BY program_workflow_id"
+queries[1,1]="all-workflow"
+
+#--Get all the workflow states
+queries[2,0]="SELECT p.program_workflow_id,p.program_Workflow_state_id,p.retired program_workflow_state_retired, p.concept_id, c.name, p.uuid program_workflow_state_uuid FROM program_workflow_state p INNER JOIN concept_name c ON p.concept_id=c.concept_id ORDER BY program_workflow_id"
+queries[2,1]="all-workflow-states"
+
+#--Get all the person attribute types
+queries[3,0]="SELECT person_attribute_type_id, name,retired,uuid FROM  person_attribute_type ORDER BY person_attribute_type_id"
+queries[3,1]="all-person-attributes"
```

Comment:
Do you think we should maybe also include relationship types?

Reply:
Did not get that, what kind of relationship?

Reply:
Well depends I guess, but on the metadata server we have the following:
![Screenshot from 2019-07-09 10-57-40](https://user-images.githubusercontent.com/19207552/60874510-710c2380-a238-11e9-8f9f-cf9c1cae03fc.png)